airflow/providers/microsoft/azure/operators/container_instances.py

```diff
@@ -151,7 +151,7 @@ def __init__(
 
         self.ci_conn_id = ci_conn_id
         self.resource_group = resource_group
-        self.name = self._check_name(name)
+        self.name = name
```

Comment:
Looks like still it doesn't work. Need to make change in the execute function as follows. 
self.name = self._check_name(self.name)

Reply:
Thanks @dirrao for your comment 😄 
I looked at it again and look it already there.
https://github.com/apache/airflow/blob/59c76cfb3036fab16cef0d70bab052db1fd7f1f9/airflow/providers/microsoft/azure/operators/container_instances.py#L189

Althogh `self._check_name` return a value 
https://github.com/apache/airflow/blob/59c76cfb3036fab16cef0d70bab052db1fd7f1f9/airflow/providers/microsoft/azure/operators/container_instances.py#L372-L381
It's look like it don't change `name` any time.

But maybe to avoid future mistakes changing to this `self.name = self._check_name(self.name)`  will be better.
What do you think?

Reply:
Presumably the check in the constructor was to fail faster, but it's completely fine to just leave the `_check_name()` call in `execute()`. Although, you don't need the first branch that checks if the `name` arg is a Jinja template. The value will be rendered _just_ before `execute()` is called so this logic will never be reached.